.github/ISSUE_TEMPLATE/epic_release.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Epic Release
+about: Epic Release Checklist
+title: 'Epic Release: [epic name]'
+labels: epic-release-activities
+assignees: ''
+
+---
+
+The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done](https://sap.github.io/spartacus-docs/definition-of-done/)
+
+## General
+
+- [ ] Epic has been tested on a production-like server using Spartacus installation script
+- [ ] Epic has been tested in all supported browsers and devices (Check list on DoD page)
+- [ ] Epic has sufficient end to end (cypress) test coverage
+- [ ] Schematics code deprecations have been provided and tested
+- [ ] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
+- [ ] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
+- [ ] Epic CSS supports directionality (if required)
+- [ ] Epic code scans have been conducted (sonar, whitesource, checkmarx)
+
+## Audits/reviews
+
+- [ ] Architecture review of the epic (with @tobi-or-not-tobi)
+- [ ] Security and threat modeling audit/review with the Security team (main contact is @giancorderoortiz)
+- [ ] Accessibility audit/review (main contact is @developpeurweb)
+
+## New Library
+
+If your epic will be introduced or introduces a new library:
+
+- [ ] Library provides schematics to install the library
+- [ ] Library modules can be lazy loaded appropriately
+- [ ] Library has been tested for a customer using Spartacus installation script
+
```

Comment:
With this checklist we are not really covering future scenarios that require modules from the SAP integration package (like we are discussing with GC at the moment. Commerce core, based on recipe 'cx' is no longer sufficient). It's only mentioned implicitly by 'Epic has sufficient end to end (cypress) test coverage'
 
Maybe we should have a section
```
## CI
- Ensure that we find a common commerce setup that can serve the entire SPA scope, including the new feature.

```

Reply:
From the architects, out of the box `cx` recipe should satisfy this. Which is what we've been doing until now.

Reply:
No, cx recipe does not cover CPQ support. That's why we are having a discussion with @giancorderoortiz around setting up commerce for this flavor.

CPQ support _would_ be covered by cx-integrations, but other integration scenarios are not part of cx-integrations yet-> I fear we need to have these considerations as an explicit step

Reply:
Ok, thanks for letting me (us) know. Would you mind adding them after we merge this? Also, can you let us know if they are documented? Thanks.